extension/src/Utils/classPathUtility.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as Configs from '../configs';
+import { Logger } from '../logger';
+
+import * as archiver from 'archiver';
+import * as fileUrl from 'file-url';
+import * as fs from 'fs';
+import * as mkdirp from 'mkdirp';
+import * as os from 'os';
+import * as path from 'path';
+
+export class ClassPathUtility {

```

Comment:
Do you think the members of this class popular in the future? If there would be only one caller, then suggest putting the functions into the caller class directly.
The processing logic is to resolve a very specific problem which the JarFileTestRunner has. So to me, it's not common enough to form a utility class.

Reply:
The methods here are to solve a general problem that class path might be too long for command line to handle, so I extract them to a utility method.